tanner/create_tables.py

```diff
@@ -0,0 +1,71 @@
+import asyncio
+
+
+class CreateTables:
+    @staticmethod
+    async def create_data_tables(pg_client):
+        """Create all the required tables in
+            the postgres database
+
+        Arguments:
+            pg_client {aiopg.pool.Pool}
+        """
+        async with pg_client.acquire() as conn:
+            async with conn.cursor() as cur:
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "paths" (
+                    "sess_uuid" UUID PRIMARY KEY NOT NULL UNIQUE,
```

Comment:
This shouldn't be a primary key but a foreign key constraint to the `sess_uuid` in the `session_data` table.
Besides that, a primary key is always not null and unique.